Dockerfile-debian.template

```diff
@@ -12,6 +12,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # Python 3.4 on Stretch+ needs to use an older version of "libssl-dev" (these lines both get removed for every other combination)
 		libssl-dev \
 		tk-dev \
+		uuid-dev \
```

Comment:
Hah, we finally found a popular `-dev` that isn't already part of `buildpack-deps`! :smile:

If we can find more examples of it being used (Ruby gems, etc), I'd be +1 to adding it to `buildpack-deps` itself, but that's a separate discussion. :innocent: